Fem: Add suppressible extension to objects - fixes #12115

marioalexis84 opened the pull request:
Fixes #12115.
@FEA-eng

FEA-eng reviewed (commented):
When right-clicking on MaterialMechanicalNonlinear objects, there's an error message ("No extension of given type available").

Could it be also added for mesh-based objects - mesh region, group and boundary layer? Maybe even Elmer equations. Solver objects themselves don't need it.

marioalexis84 commented:
> When right-clicking on MaterialMechanicalNonlinear objects...

Fixed.

> Could it be also added for mesh-based objects - mesh region, group and boundary layer?

Yes, but it requires a bit more care. Let's merge this one and then prepare another pull request.

FEA-eng commented:
> Yes, but it requires a bit more care. Let's merge this one and then prepare another pull request.

Ok, should I create a new issue for it?

marioalexis84 commented:
@FEA-eng  Suppressible extension added to mesh region, group and boundary layer

FEA-eng commented:
> @FEA-eng Suppressible extension added to mesh region, group and boundary layer

Great, thanks. Works properly.
